jobs/signing/sign-artifacts/Jenkinsfile

```diff
@@ -262,6 +265,13 @@ node {
                     }
                 }
             }
+
+            echo "Archiving artifacts in jenkins:"
```

Comment:
we would likely want these in a `finally {}` block in case there's partial success (one sig pushes, another doesn't).

Reply:
ack